index.html

```diff
@@ -12,8 +15,78 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
-
-<body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+<div class="full-container">
+  <body>
+    <div class="container">
+        <nav>
+          <a href="">Services</a>
+          <a href="">Product</a> 
+          <a href="">Vision</a>
+          <a href="">Features</a>
+          <a href="">About</a>
+          <a  href="">Contact</a>
+        </nav>
+      <div class="logo" >
```

Comment:
Way to go wrapping the logo in a div. It makes using the box model on the image easier. Also it allows you to add a border-bottom in mobile view.